editor.planx.uk/src/@planx/components/shared/Preview/SummaryList.tsx

```diff
@@ -618,6 +620,25 @@ function FileUploadAndLabel(props: ComponentProps) {
   );
 }
 
+function Page(props: ComponentProps) {
+  const answers = getAnswersByNode(props) as SchemaUserResponse[];
+  const fields = (props.node.data as Page).schema.fields;
+
+  return (
+    <>
+      <Box component="dt">{props.node.data.title}</Box>
+      <Box component="dd" sx={{ gap: 2, display: "flex", flexDirection: "column" }}>
+        {fields.map((field) => (
+          <Box>
+            <Typography fontWeight={FONT_WEIGHT_SEMI_BOLD}>{field.data.title}</Typography>
+            <Typography>{answers[0][field.data.fn]}</Typography>
```

Comment:
nit: I think this needs to account for a `"map"` type field separately - either by showing a static map or a simple summary (eg "2 features") ??

Reply:
Nice catch! Fixed in [11aa28a](https://github.com/theopensystemslab/planx-new/pull/3679/commits/11aa28ae166ea1506ae6e1e806f6a887ecd69fea)

Reply:
Perfect, thanks !